ci/dev/test-unit.sh

```diff
@@ -30,7 +30,7 @@ main() {
   # We must keep jest in a sub-directory. See ../../test/package.json for more
   # information. We must also run it from the root otherwise coverage will not
   # include our source files.
-  CS_DISABLE_PLUGINS=true ./test/node_modules/.bin/jest "$@"
+  CS_DISABLE_PLUGINS=true ./test/node_modules/.bin/jest "$@" --testRegex "./test/unit/.*ts" --testPathIgnorePatterns "./test/unit/node/test-plugin"
```

Comment:
NOTE: since we run `jest` from the root, it grabs the `package.json.jest` config and uses that for running tests. By adding these flags here, we can use the same Jest config for both unit & integration tests and then pass additional flags as needed.